### ISSUE TITLE
Can mmseg4j-all be used from mavenCentral?

### ISSUE THREAD

**Issue by ghost** [AndBible/and-bible#21]
Could the jar be replaced by one of those?

http://search.maven.org/#artifactdetails|com.chenlb.mmseg4j|mmseg4j-solr|2.2.0|jar
http://search.maven.org/#artifactdetails|com.chenlb.mmseg4j|mmseg4j-core|1.10.0|jar
http://search.maven.org/#artifactdetails|com.chenlb.mmseg4j|mmseg4j-analysis|1.9.1|jar

**Comment by ildar** [comment]
@krt16s , I am sorry, didn't quite catch it: is this still an issue?

**Comment by krt16s** [comment]
Yes, the problem still exists. However, fdroid is lacking recent vesions due to an other issue...  gradle doesnt look valid and we are hitting a bug in the gradle plugin.. I have it building locally again, but I have to find some spare time to clean things up and apply to fdroid...

**Comment by timbze** [comment]
Outdated and already done
